packages/solo/src/vat-http.js

```diff
@@ -97,6 +104,14 @@ export function buildRootObject(vatPowers) {
     urlToHandler.set(url, commandHandler);
   }
 
+  const [resolveWallet, resolveAttMaker] = introduceWith(
+    async (wallet, attMaker) => {
+      const walletAdmin = await E(wallet).getAdminFacet();
+      // console.debug('introduce', { wallet, walletAdmin, attMaker });
+      E(walletAdmin).resolveAttMaker(attMaker);
```

Comment:
Ah didn't expect this method to be useful still, cool!

Reply:
I was hoping / planning to not use it, but I couldn't figure out an alternative.